packages/auth/src/OAuth.ts

```diff
@@ -107,7 +107,8 @@ class OAuth {
       return null;
     }
 
-    return new Credentials('roman-carto', this._carto.accessToken.raw);
+    // TODO: Fix or remove!!!
+    return new Credentials(this._carto.userInfo, this._carto.accessToken.raw);
```

Comment:
Wow, we had a strange const there hardcoded 😄 

Is the comment still relevant. I mean, userInfo is enough to fix it?